cloud/clusterapi.go

```diff
@@ -233,19 +231,19 @@ func (ca *ClusterApi) CreatePharmerSecret() error {
 	}
 
 	if err = CreateSecret(ca.kc, "pharmer-certificate", ca.namespace, map[string][]byte{
```

Comment:
required only for do, linode and packets. we should not create these secrets for all other providers